tokens/theme-m3-typography.json

```diff
@@ -0,0 +1,116 @@
+{
+  "display": {
+    "large": {
+      "value": {
+        "fontFamily": "Roboto",
+        "fontWeight": "Regular",
+        "lineHeight": "64",
+        "fontSize": "57",
+        "letterSpacing": "-0.25"
```

Comment:
where are all those hardcoded values coming from? 
Are they the Material 3 default? If so how do we keep that in sync when they will be updated on Material's side?

Reply:
There's no way to automate those. Those are Material 3 typography tokens that don't have a counterpart in Compound but might be used (many of them very rarely) in Material 3 components. We'll just manually update them if needed.

Reply:
I'm not saying there is a way to automate  those, i'm wondering where they come from and whether we can link in the `description` field where they were taken from in case we need to update that in the future. 

otherwise makes sense